src/components/interlocutor/screens/InterlocutorAdd/containers/InterlocutorAddContainer.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import 'moment/locale/fr';
+moment.locale('fr');
+
+import InterlocutorAddView from '../views/InterlocutorAddView';
+import { addInterlocutor } from '../actions/index';
+
+
+class InterlocutorAddContainer extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state = {
+      interlocutor: {
+        name: "",
+        link_to_me: "",
+        date: moment(),
+        key: this.props.lastID + 1,
+      },
+      isFormValid: false,
+    }
+    this.update = this.update.bind(this)
+    this.validateForm = this.validateForm.bind(this)
+    this.setName = this.setName.bind(this)
+    this.setLinkToMe = this.setLinkToMe.bind(this)
+  }
+
+  validateForm(){
+    isFormValid = true;
+    isFormValid = isFormValid && this.state.interlocutor.name.length > 0
+    isFormValid = isFormValid && this.state.interlocutor.link_to_me.length > 0
+    this.setState({isFormValid: isFormValid})
+    }
+  
+  setName(name){
+    this.setState({...this.state, interlocutor: {...this.state.interlocutor, name: name}}, () => {this.validateForm; this.update()})
+  }
+
+  setLinkToMe(link_to_me){
```

Comment:
case is strange here, I think `link_to_me` should be renamed `linkToMe`. Same in the local state.